app/controllers/user_api_keys_controller.rb

```diff
@@ -47,6 +45,7 @@ def new
     @push_url = params[:push_url]
     @localized_scopes = params[:scopes].split(",").map { |s| I18n.t("user_api_key.scopes.#{s}") }
     @scopes = params[:scopes]
+    @padding = params[:padding]
```

Comment:
padding can be _anything_ now? Would it make sense to limit what values it can be?

Reply:
Ok, I've added validation and refactored the controller's require and validate params a little bit